attiny/attiny.ino

```diff
@@ -50,13 +51,15 @@ enum State {
   PI_WDT_FAILED             // WDT for RPi failed. Turn off and on.
 };
 
-#define I2C_READ_REG_LEN 1
-#define I2C_READ_BATTERY_VOLTAGE 0x20
+#define I2C_READ_REG_LEN 2
+#define I2C_READ_BATTERY_VOLTAGE_1 0x20
+#define I2C_READ_BATTERY_VOLTAGE_2 0x21
```

Comment:
Would better names be `I2C_READ_BATTERY_VOLTAGE_LO` and `I2C_READ_BATTERY_VOLTAGE_HI` to make it clear which byte is being returned for each request?